tools/nemo_forced_aligner/utils/data_prep.py

```diff
@@ -14,12 +14,25 @@
 
 import json
 import os
+from dataclasses import dataclass, field
+from pathlib import Path
+from typing import List, Union
 
-import soundfile as sf
+import librosa
 import torch
```

Comment:
Is it necessary to switch to librosav?

Reply:
I think that was an unused variable.
I'm using librosa now to get the duration of the audio file. I chose it arbitrarily, could also use soundfile. Do you know if one is better than the other?

Reply:
Both are fine.